docs/styleguides/python.md

```diff
@@ -0,0 +1,173 @@
+# DCOS Python Style Guide
+###### Version 0.1
+###### April-6-2017
+
+For Python source code within DCOS, we prefer to keep Google Python Style Guide Revision 2.59 as the base and **extend** it with
```

Comment:
That's what I've tried. That's the reason this style guide is small. :) If you see any repetition (that can be avoided), please point out. :)